### DISCUSSION TITLE
CMU 15-213: CSAPP - CSDIY.wiki

### DISCUSSION THREAD

**Discussion by giscus** [PKUFlyingPig/cs-self-learning#537] [Announcements]
# CMU 15-213: CSAPP - CSDIY.wiki

CS自学指南

https://csdiy.wiki/en/%E4%BD%93%E7%B3%BB%E7%BB%93%E6%9E%84/CSAPP/?h=cmu

**Comment by gerrybooter** [comment]
居然没人？！

**Reply by remyhuang03** [reply]
CSAPP的书很赞，但是课没上过qwq，感觉看书就已经能学到不少了

**Reply by Pulp-fiction-42** [reply]
hello，现在还在学习吗？我最近也准备开启了

**Reply by remyhuang03** [reply]
> hello，现在还在学习吗？我最近也准备开启了

已经在看最后一章快要看完一遍了。我大概是从去年暑假开始利用假期时间读的，今年是读CSAPP的第三个假期，多数内容值得学习。

**Comment by Pulp-fiction-42** [comment]
On Thu, Jul 31, 2025 at 21:56 Remy Huang ***@***.***> wrote:

> hello，现在还在学习吗？我最近也准备开启了
>
> 已经在看最后一章快要看完一遍了。我大概是从去年暑假开始利用假期时间读的，今年是读CSAPP的第三个假期，多数内容值得学习。
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/PKUFlyingPig/cs-self-learning/discussions/537#discussioncomment-13947979>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BQQCRXGCX6VXRM6MYVKEAR33LIOAVAVCNFSM6AAAAACCPEXBA6VHI2DSMVQWIX3LMV43URDJONRXK43TNFXW4Q3PNVWWK3TUHMYTGOJUG44TOOI>
> .
> You are receiving this because you commented.Message ID:
> <PKUFlyingPig/cs-self-learning/repo-discussions/537/comments/13947979@
> github.com>
>
好的明白，我今年研究生一年级，非科班cs目前在从事ai方面的工作，也想系统性学习下cs，就打算通过csapp了。从去年暑假一直看到今年暑假感觉学习曲线还是挺长的，现在那些需要动手的lab请问还开放吗。
祝学习顺利

**Reply by remyhuang03** [reply]
> On Thu, Jul 31, 2025 at 21:56 Remy Huang ***@***.***> wrote: hello，现在还在学习吗？我最近也准备开启了 已经在看最后一章快要看完一遍了。我大概是从去年暑假开始利用假期时间读的，今年是读CSAPP的第三个假期，多数内容值得学习。 — Reply to this email directly, view it on GitHub <[#537 (reply in thread)](https://github.com/PKUFlyingPig/cs-self-learning/discussions/537#discussioncomment-13947979)>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/BQQCRXGCX6VXRM6MYVKEAR33LIOAVAVCNFSM6AAAAACCPEXBA6VHI2DSMVQWIX3LMV43URDJONRXK43TNFXW4Q3PNVWWK3TUHMYTGOJUG44TOOI> . You are receiving this because you commented.Message ID: <PKUFlyingPig/cs-self-learning/repo-discussions/537/comments/13947979@ github.com>
> 好的明白，我今年研究生一年级，非科班cs目前在从事ai方面的工作，也想系统性学习下cs，就打算通过csapp了。从去年暑假一直看到今年暑假感觉学习曲线还是挺长的，现在那些需要动手的lab请问还开放吗。 祝学习顺利

对于 labs 的话，可以在 [CSAPP 官方网站](https://csapp.cs.cmu.edu/) 的 [Lab Assignments 页面](https://csapp.cs.cmu.edu/3e/labs.html) 找到，但是没有提供答案。

如果是章末的 exercises 的话，这个 [第三方网站](https://dreamanddead.github.io/CSAPP-3e-Solutions/) 可以提供答案参考。

祝学习顺利，一起加油呀！
